internals/secrethub/generate.go

```diff
@@ -80,12 +99,66 @@ func (cmd *GenerateSecretCommand) run() error {
 
 	fmt.Fprint(cmd.io.Stdout(), "Writing secret value...\n")
```

Comment:
I'm seeing this now, your call to pick it up or not, but why are we printing out `generating...writing...write complete`? If we want to print anything out, would `generating... generate complete` be sufficient?

Reply:
As this is unrelated to this PR, I've created #173 to pick this up separately.